src/DependencyRewriterPlugin.php

```diff
@@ -4,7 +4,10 @@
 
 namespace Laminas\DependencyPlugin;
 
-/** @deprecated */
+/**
+ * @deprecated
+ * @psalm-suppress UnusedClass keep the deprecation
```

Comment:
I think psalm baseline can be regenerated, and cs can be fixed with `vendor/bin/phpcbf`